automation/issuer/bot/issuer_bot.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import os
+
+import dazl
+from dazl import exercise
+
+
+dazl.setup_default_logger(logging.INFO)
+
+
+class MARKETPLACE:
+    RegisteredCustodian = 'Marketplace.Registry:RegisteredCustodian'
+    RegisteredExchange = 'Marketplace.Registry:RegisteredExchange'
+    RegisteredInvestor = 'Marketplace.Registry:RegisteredInvestor'
+    Token = 'Marketplace.Token:Token'
+
+
+def main():
+    url = os.getenv('DAML_LEDGER_URL')
+    issuer = os.getenv('DAML_LEDGER_PARTY')
+    public = os.getenv('DABL_PUBLIC_PARTY')
+
+    issuer_party = "Issuer" if not issuer else issuer
+    public_party = "Public" if not public else public
+
+    network = dazl.Network()
+    network.set_config(url=url, party_groups=[public_party])
+
+    logging.info(f'Integration will run under party: {issuer_party}')
+
+    client = network.aio_party(issuer_party)
+
+    @client.ledger_ready()
+    def say_hello(event):
+        logging.info(f"DA Marketplace Issuer bot is ready!")
+
+
+    @client.ledger_created(MARKETPLACE.RegisteredCustodian)
+    def handle_handle_registered_custodian(event):
```

Comment:
not a big deal, but just curious - why do all of these start with `handle_handle`?

Reply:
lol I'll remove. bad copy pasta

Reply:
will merge this and address on the next pr